File: 프로그래머스/1/12982. 예산/예산.py
def solution(d, budget):
    answer = 0
    d.sort()
    every = True
    for i in range(len(d)):
        budget -= d[i]
        if budget < 0:
            answer = i
            every = False
            break
    if every:
        answer = len(d)
    return answer

File: 프로그래머스/1/12982. 예산/예산.java
import java.util.Arrays;
class Solution {
    public int solution(int[] d, int budget) {
        int answer = 0;
        Arrays.sort(d);
        boolean every = true;
        
        for (int i = 0; i < d.length; i++) {
            budget -= d[i];
            if (budget < 0) {
                return i;
            }
        }
        return d.length;
    }
}